ext/app/electron/utils.ts

```diff
@@ -0,0 +1,22 @@
+import * as fse from "fs-extra";
```

Comment:
Could rename `utils.ts` to `fileUtils.ts` to reduce the scope of things that could be in this file, and it make it easier to find things here.